model.py

```diff
@@ -460,7 +460,7 @@ def forward(self, audio):
                     spectrograms[sample, t],
                     mix_stft[sample, t],
                     self.niter,
-                    use_softmask=False,
+                    softmask=self.softmask,
```

Comment:
here, we were always using the mixture phase.
I'm actually ok with this (this would mean removing the `softmask` parameter from `Separator` altogether), but I reintroduced it here for consistency